### ISSUE TITLE
Alerting: Alert Duration Wrapping in Alert List

### ISSUE THREAD

**Issue by acaos** [grafana/grafana#73349]
### What happened?

Previously, the alert list never wrapped the column containing the alert state and duration. As of v10 (I am unsure which particular revision), the alert list now sometimes wraps cells in this column, making the alert list more difficult to read.

![image](https://github.com/grafana/grafana/assets/295044/5268a218-4900-4ff1-9a06-73a4b1c4d0d1)

### What did you expect to happen?

The alert list to keep the old wrapping behavior.

Setting it to always wrap would be (in my opinion) even worse than the current behavior, where at least a good part of the time it does not wrap.

### Did this work before?

This worked in v9. I'm unsure if it broke immediately in v10 or in the update to v10.0.3.

### How do we reproduce it?

1. Create and trigger several alerts.
2. Have those alerts continue alerting for a long period of time (usually weeks or months are needed). Have other alerts that have only been triggering for seconds or minutes.
3. Look at the State column of the alert list and see that the column sometimes wraps the alert duration.

### Is the bug inside a dashboard panel?

No, this is inside the main Alert List.

### Environment (with versions)?

Grafana: v10.0.3
OS: Windows 10
Browser: Chromium

### Grafana platform?

Kubernetes

**Comment by joel113** [comment]
Tried to reproduce the issue as I was looking for an issue were I can contribute something. While it was possible to reproduce the wrapping behavior using Grafana 10.0.3, as the `flex-flow: wrap` stylesheet property is added to the `div` which encapsulates the alert state:

<img width="1667" alt="reproduced" src="https://github.com/grafana/grafana/assets/11200009/e334654e-7d7d-4b45-a999-1fbcdb18412b">

However, it was not possible to preduce in the current Grafana version (10.2.2):

<img width="1647" alt="notreproduced" src="https://github.com/grafana/grafana/assets/11200009/12f084fd-330b-4c33-b280-54fb5029e333">

There were several redesign after this issue was filed. Hence, I assume the issue is resolved.

**Comment by joel113** [comment]
You are right. On Grafana 10.2.2 it is reproducible. I should have been more precise. In my earlier post I was not checking Grafana 10.2.2, but the latest Grafana version available at the main branch, which is - I guess - going to be released with 10.3.x.

In the meantime I got a better understanding which changes did have an affect that the wrapping is no longer applied:

The react component RuleState.tsx is rendering the AlertState as well as the for time as part of the RulesTable.tsx. While the implementation in RuleState.tsx did not change, there is a difference between Grafana 10.2.2 and the main branch which Stack-Component is used:

```
<Stack gap={1}>
  <AlertStateTag state={promRule.state} isPaused={isPaused} />
  {forTime}
</Stack>
```

The [RuleState.tsx of Grafana 10.2.2](https://github.com/grafana/grafana/blob/64ee42d01ef779647e04fc88aa46673db38773f3/public/app/features/alerting/unified/components/rules/RuleState.tsx) is using the [Stack component from the Grafana Experimental package](https://github.com/grafana/grafana-experimental/blob/main/src/QueryEditor/Stack.tsx), which has the [default behavior of wrapping if no wrapping-property is passed](https://github.com/grafana/grafana-experimental/blob/main/src/QueryEditor/Stack.tsx#L25):

```
 flexWrap: props.wrap ?? true ? 'wrap' : undefined,
```

The [RuleState.tsx of Grafana current main branch](https://github.com/grafana/grafana/blob/25779bb6e5cedef712db15b9cc77ab4af68c31dd/public/app/features/alerting/unified/components/rules/RuleState.tsx) is using the [Stack component from Grafana UI](https://github.com/grafana/grafana/blob/e286a3a652a2e66240d15f320065a08f6a62aa9c/packages/grafana-ui/src/components/Layout/Stack/Stack.tsx), which does not have the default behavior of wrapping. As in [RuleState.tsx](https://github.com/grafana/grafana/blob/25779bb6e5cedef712db15b9cc77ab4af68c31dd/public/app/features/alerting/unified/components/rules/RuleState.tsx) only the `gap` property is passed, the `wrap` property is not used anymore. According to my understanding, the changes were introduced with the following pull request: https://github.com/grafana/grafana/commit/25779bb6e5cedef712db15b9cc77ab4af68c31dd.
